### DISCUSSION TITLE
How did you do the landing page?

### DISCUSSION THREAD

**Discussion by naquiroz** [sniptt-official/snip-cli#11] [General]
Hi! I'm just curious, what frameworks and component libraries were used to make the website https://www.sniptt.com/?
I really like its design, and I'm trying to do something similar.

**Comment by rickymarcon** [comment]
Hi @naquiroz - thanks for the question!

We are currently using [Docusaurus](https://docusaurus.io/) for https://snippt.com. Since we don't need many of your typical input components, the component library is completely custom for now, however, this will likely change in the future as the site grows. The API docs are powered by [Redoc](https://github.com/Redocly/redoc). 

Not much else to it!

**Reply by naquiroz** [reply]
Wow! Amazing, thanks so much!
